sunpy/net/jsoc/jsoc.py

```diff
@@ -438,7 +438,10 @@ def fetch(self, jsoc_response, path=None, progress=True, overwrite=False,
         time.sleep(sleep/2.)
 
         for response in responses:
-            response.wait(verbose=progress)
+            if Version(drms.__version__) >= Version('0.7.0'):
```

Comment:
So now we don't get messages with the export status any more?

Reply:
You get via logging.info, no keyword anymore.

Reply:
Seeems sensible to me and is consistent with how we control output in the rest of the package.